Makefile

```diff
@@ -7,11 +7,11 @@ assemble:
 
 .PHONY: build
 build:
-	./gradlew build -x :smoke-tests:test --stacktrace
+	./gradlew build -x :smoke-tests:test --debug --stacktrace
```

Comment:
is this needed? I don't like to run debug locally. As below in the `smoke-test` target

Reply:
have removed the debug option